microprofile/server/src/main/java/io/helidon/microprofile/server/JaxRsService.java

```diff
@@ -72,7 +72,10 @@
 import static org.glassfish.jersey.CommonProperties.PROVIDER_DEFAULT_DISABLE;
 import static org.glassfish.jersey.server.ServerProperties.WADL_FEATURE_DISABLE;
 
-class JaxRsService implements HttpService {
+/**
+ * JAX-RS Service.
+ */
+public class JaxRsService implements HttpService {
```

Comment:
This class should not be public. If needed, put the test in the same package (with a not explaining that we need to test package local class.

Reply:
Fixed